apps/address-service/domains/common/utils/services/suggest/providers/PullentiSuggestionProvider.js

```
@@ -0,0 +1,63 @@
+    async get ({ query, context = '', count = SEARCH_COEF_THRESHOLD, helpers = {} }) {
+        const xmlResult = await this.client.searchByText(query, { count: count || DEFAULT_COUNT })
+        const jsonResult = this.xmlParser.parse(xmlResult)
+        if (!jsonResult?.searchresult?.gar) {
+            return []
+        }
+
+        let gars = jsonResult?.searchresult?.gar || []
+        gars = Array.isArray(gars) ? gars : [gars]
+
+        const xmlResults = await Promise.all(gars.map((gar) => {
+            return gar && gar.path ? this.client.searchByAddress(gar.path) : null
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)